packages/relay/src/TransactionManager.ts

```diff
@@ -68,41 +66,47 @@ export class TransactionManager {
     this.nonces[this.workersKeyManager.getAddress(0)] = 0
   }
 
-  async _init (): Promise<void> {
+  async _init (transactionType: TransactionType): Promise<void> {
+    this.transactionType = transactionType
     this.rawTxOptions = this.contractInteractor.getRawTxOptions()
     if (this.rawTxOptions == null) {
       throw new Error('_init failed for TransactionManager, was ContractInteractor properly initialized?')
     }
   }
 
-  printBoostedTransactionLog (txHash: string, creationBlockNumber: number, gasPrice: number, isMaxGasPriceReached: boolean): void {
-    const gasPriceHumanReadableOld: string = new EthVal(gasPrice).toGwei().toFixed(4)
+  printBoostedTransactionLog (txHash: string, creationBlockNumber: number, maxFeePerGas: number, maxPriorityFeePerGas: number, isMaxGasPriceReached: boolean): void {
+    const maxFeePerGasHumanReadableOld: string = new EthVal(maxFeePerGas).toGwei().toFixed(4)
+    const maxPriorityFeePerGasHumanReadableOld: string = new EthVal(maxPriorityFeePerGas).toGwei().toFixed(4)
     this.logger.info(`Boosting stale transaction:
-hash         | ${txHash}
-gasPrice     | ${gasPrice} (${gasPriceHumanReadableOld} gwei) ${isMaxGasPriceReached ? chalk.red('(MAX GAS PRICE REACHED)') : ''}
-created at   | block #${creationBlockNumber}
+hash                     | ${txHash}
+maxFeePerGas             | ${maxFeePerGas} (${maxFeePerGasHumanReadableOld} gwei) ${isMaxGasPriceReached ? chalk.red('(MAX GAS PRICE REACHED)') : ''}
```

Comment:
What do you mean by "how"? It displays maxFeePerGas which is equal to gasPrice on legacy networks in our case. I can add a check whether we're on legacy, then print "gasPrice" instead, but since we do want to remove legacy support eventually, I didn't think it was necessary.